src/main/java/com/chaosthedude/naturescompass/NaturesCompass.java

```diff
@@ -10,10 +10,7 @@
 import com.chaosthedude.naturescompass.config.ConfigHandler;
 import com.chaosthedude.naturescompass.gui.GuiHandler;
 import com.chaosthedude.naturescompass.items.ItemNaturesCompass;
-import com.chaosthedude.naturescompass.network.PacketCompassSearch;
-import com.chaosthedude.naturescompass.network.PacketRequestSync;
-import com.chaosthedude.naturescompass.network.PacketSync;
-import com.chaosthedude.naturescompass.network.PacketTeleport;
+import com.chaosthedude.naturescompass.network.*;
```

Comment:
please revert it. no star imports. thanks

Reply:
Apologies, IntelliJ autoformatting